### PR TITLE
FPLA-3344:  Welsh C11a enhanced translation

### PR DESCRIPTION
**Pull request by samuelzerozero**
https://tools.hmcts.net/jira/browse/FPLA-3344
Welsh C11a enhanced translation

C110A needs to be the real template (see attachments) + docmosis unit/integration tests

In the service there are still WELSH parts of the C110a Welsh version of document that are generated dynamically by the service. Some of them need translation in order to have a fully version of the document.

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java [hunk 1]
```diff
@@ -552,172 +609,189 @@ private String getDefaultIfNullOrEmpty(final Optional<String> value) {
...
     }
 
-    private void formatChildLivingSituationDisplay(final ChildParty child, final StringBuilder sb) {
+    private void formatChildLivingSituationDisplay(final ChildParty child, final StringBuilder sb,
+                                                   Language applicationLanguage) {
         switch (fromString(child.getLivingSituation())) {
             case HOSPITAL_SOON_TO_BE_DISCHARGED:
                 if (child.getDischargeDate() != null) {
-                    sb.append("Discharge date: ").append(formatDateDisplay(child.getDischargeDate()));
+                    sb.append("Discharge date: ")
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:636]: Is this getting translated?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java [hunk 2]
```diff
@@ -552,172 +609,189 @@ private String getDefaultIfNullOrEmpty(final Optional<String> value) {
...
+                        .append(formatDateDisplay(child.getDischargeDate(), applicationLanguage));
                 }
                 break;
             case REMOVED_BY_POLICE_POWER_ENDS:
                 if (child.getDatePowersEnd() != null) {
-                    sb.append("Date powers end: ").append(formatDateDisplay(child.getDatePowersEnd()));
+                    sb.append("Date powers end: ")
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:642]: Is this getting translated?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java [hunk 3]
```diff
@@ -552,172 +609,189 @@ private String getDefaultIfNullOrEmpty(final Optional<String> value) {
...
+                        .append(formatDateDisplay(child.getDatePowersEnd(), applicationLanguage));
                 }
                 break;
             case VOLUNTARILY_SECTION_CARE_ORDER:
                 if (child.getCareStartDate() != null) {
-                    sb.append("Date this began: ").append(formatDateDisplay(child.getCareStartDate()));
+                    sb.append("Date this began: ")
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:648]: Is this getting translated?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java [hunk 4]
```diff
@@ -552,172 +609,189 @@ private String getDefaultIfNullOrEmpty(final Optional<String> value) {
...
+                        .append(formatDateDisplay(child.getCareStartDate(), applicationLanguage));
                 }
                 break;
             default:
                 if (child.getAddressChangeDate() != null) {
                     sb.append("Date this began: ")
```
Comment (lewisbirks) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:654]: Is this getting translated?

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java (hunk 1 above)
Comment (TSeelig) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:636]: To be translated in [DFPL-255](https://tools.hmcts.net/jira/browse/DFPL-255).

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java (hunk 2 above)
Comment (TSeelig) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:642]: To be translated in [DFPL-255](https://tools.hmcts.net/jira/browse/DFPL-255).

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java (hunk 3 above)
Comment (TSeelig) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:648]: To be translated in [DFPL-255](https://tools.hmcts.net/jira/browse/DFPL-255).

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java (hunk 4 above)
Comment (TSeelig) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionGenerationService.java:654]: To be translated in [DFPL-255](https://tools.hmcts.net/jira/browse/DFPL-255).

**Review by lewisbirks — APPROVED** [review]

**Review by meganaslattery — APPROVED** [review]

**Pull request merged**